README.md

```diff
@@ -252,7 +252,7 @@ repositories {
 }
 dependencies {
     // Koin testing tools
-    testcompile 'org.koin:koin-test:{{ site.current_version }}'
+    testCompile 'org.koin:koin-test:{{ site.current_version }}'
```

Comment:
This should probably just be changed to `testImplementation` to conform to the new Gradle configurations?

Reply:
Yes, fix it directly with `testImplementation`